collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java

```diff
@@ -53,6 +53,8 @@
     String statusMessage;
 
     private FormsDao mFormsDao;
+    
+    private List<String> idsToDelete;
```

Comment:
Variables should be as tightly scoped as possible. I believe `idsToDelete` is only used in `doInBackground` so it should be declared there.